examples/market_maker.py

```diff
@@ -0,0 +1,19 @@
+from ucf_exchange_client import Strategy
+
+market_maker = Strategy('mm')
```

Comment:
So as-written this strategy doesn't actually issue any trades. You should define a method that handles `Book` or something and actually does like `yield Buy(ticker, price, quantity)`.